assets/javascripts/tests.js

```diff
@@ -57,6 +57,8 @@ function renderTestName(data, type, row) {
 
   var html = '';
   if (is_operator) {
+    html += '<a class="copy-jobid" href="' + urlWithBase('/tests/' + row.id) + '" data-jobidi="' + row.id + '">';
+    html += '<i class="action fa fa-fw fa-copy" title="Copy job id"></i></a>';
```

Comment:
This is the full "job URL", not just ID, right?

Reply:
Yes, the `<a href=` points to the full job id but if you're javascript is enabled then it will not be followed.
That was suggested by @asdil12 [here](https://github.com/os-autoinst/openQA/pull/5249#discussion_r1263890961).

Reply:
I am not sure I understand. But if the end result is the full job URL then please call it as such in code as well as in the help and git commit message

Reply:
In the `href=""` is the full test URL in case your javascript is off.
But if the javascript is working then only the job id is put to your clipboard.

Reply:
If you need job URL it's two clicks to copy it. But if you f.e. want to clone 10 different jobs, then it is very time consuming to copy the job URL, parse the job ID, store the job ID and repeat 10 times. So this button only gets you the job id.

Reply:
Also people usually reuse commands from their history, so for example `openqa-clone-job https://openqa.opensuse.org/tests/11560397 --host https://pdostal-server.suse.cz --show-progress --skip-chained-deps` and if I want to do 5 verification runs it's usually only the job ID to replace.

Reply:
Yeah, sorry, I don't think that's a use case I would like feature *more* prominently to just copy the ID instead of the URL as that sounds like a rather personal choice of how you process command lines. Instead what I suggest to do commonly is to iterate over full job URLs, something like:

```bash
for i in https://openqa.opensuse.org/tests/11560397 https://openqa.opensuse.org/tests/11560234 https://openqa.opensuse.org/tests/11560402; do
openqa-clone-job $i --host https://pdostal-server.suse.cz --show-progress --skip-chained-deps
done
```

Reply:
Your command work when you want to clone bunch of jobs at once but if I want to clone only one job and next job in half an hour you end up replacing the URL over and over.

Then let's say I want to monitor a job in my command line: `openqa-cli api --osd --pretty jobs/$job_id`
Oh and then I have review and I want to use the awesome [grisu48/openqa-mon](https://github.com/grisu48/openqa-mon): `openqa-mon http://openqa.opensuse.org -j $ID1,$ID2,$ID3,...`

I've so far received quite positive feedback.
If the icon is in wrong place, we can move it.

Basically every system dealing with jobs or tickets displays the ID somewhere. And I think there's a reason for that. See:
![Screenshot 2023-07-18 at 11 48 28](https://github.com/os-autoinst/openQA/assets/1254493/7aa5e02a-df4e-40ed-84c2-1e036593e697)
![Screenshot 2023-07-18 at 11 46 40](https://github.com/os-autoinst/openQA/assets/1254493/e20e6a11-a33d-432b-94ed-50d0f1022412)
![Screenshot 2023-07-18 at 11 44 58](https://github.com/os-autoinst/openQA/assets/1254493/41746486-910d-4260-8b5f-1845e41ef3da)
![Screenshot 2023-07-18 at 11 44 03](https://github.com/os-autoinst/openQA/assets/1254493/4b6cddfd-b03c-4d35-8bf9-ea4ea639b93a)
![Screenshot 2023-07-18 at 12 01 32](https://github.com/os-autoinst/openQA/assets/1254493/b371ea23-1a04-47dd-8aa6-d3bc7c280810)

Reply:
I also think this feature might turn out to be useful to more folk than just Pavel :slightly_smiling_face:

Reply:
Ok, convinced :D